src/Elastic.Apm/Api/ISpan.cs

```diff
@@ -46,10 +46,22 @@ public interface ISpan
 		/// </summary>
 		Dictionary<string, string> Tags { get; }
 
+		/// <summary>
+		/// Offset relative to the transaction's timestamp identifying the start of the span, in milliseconds.
+		/// </summary>
+		//decimal Start { get; set; }
+
+		long Timestamp { get; }
+
+		/// <summary>
+		/// Hex encoded 128 random bits ID of the correlated trace.
+		/// </summary>
+		string TraceId { get; }
+
 		/// <summary>
 		/// UUID of the enclosing transaction.
 		/// </summary>
-		Guid TransactionId { get; }
+		string TransactionId { get; }
```

Comment:
It isn't really a `Guid`.  When I used `Guid`, it did not work with the server once I switched to V2, and that is probably, because by default there are `-` characters in the `Guid`, which the server does not accept. Also a `Guid` is 128bit, but the `TransactionId` is 64bit. This shouldn't have worked with V1 either I think. Btw. I just see that the `summary` is outdated, I'll update that.